koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -945,3 +945,404 @@ def handler(self, src, target, opts=None):
             'repositories': all_repositories,
             'koji_builds': all_koji_builds,
         }
+
+
+class BuildSourceContainerTask(BaseTaskHandler):
+    # Start builds via osbs
+    Methods = ['buildSourceContainer']
+    # Same value as for regular 'build' method.
+    _taskWeight = 2.0
+
+    # JSON Schema definition for koji buildSourceContainer task parameters
+    # Used to validate arguments passed to the handler() method of this class
+    PARAMS_SCHEMA = {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "description": "Parameters for a koji buildSourceContainer task.",
+
+        "type": "array",
+        "items": [
+            {
+                "type": "string",
+                "description": "Build target."
+            },
+            {
+                "type": ["object", "null"],
+                "properties": {
+                    "scratch": {
+                        "type": "boolean",
+                        "description": "Perform a scratch build?"
+                    },
+                    "koji_build_id": {
+                        "type": ["string", "null"],
```

Comment:
Shouldn't this be an integer? (and null is definitely not valid)
```suggestion
                        "type": "integer",
```

Reply:
ok